src/main/java/pl/winiarczyk/Hall.java

```diff
@@ -0,0 +1,14 @@
+package pl.winiarczyk;
+
+public class Hall {
+
+    final int seatNumber;
+    final int rowNumber;
+    boolean busy;
```

Comment:
Te pola powinny być 'private'. W Javie prawie nigdy nie spotkasz sytuacji gdy do pól można dobrać się bezpośrednio, bo może być to niebezpieczne. 

https://1024kb.pl/nauka-java/hermetyzacja-java/

![image](https://user-images.githubusercontent.com/17989854/131245308-33ff0746-6700-454f-84b4-512dbb38cffb.png)